test/norm_num.lean

```diff
@@ -219,6 +220,24 @@ section norm_num_cmd_variable
 
 end norm_num_cmd_variable
 
+namespace norm_num_erase
+
+example : 3 ^ 3 + 4 = 31 := by norm_num1; with_reducible rfl
+
+attribute [-norm_num] Mathlib.Meta.NormNum.evalPow
+example : 3 ^ 3 + 4 = 31 := by
+  norm_num1
+  guard_target =ₛ 3 ^ 3 + 4 = 31
+  rfl
+
+/- should error: 'Mathlib.Meta.NormNum.evalPow' does not have [norm_num] attribute -/
+/-
+attribute [-norm_num] Mathlib.Meta.NormNum.evalPow
+attribute [-norm_num] Mathlib.Meta.NormNum.evalPow in
```

Comment:
Could you clarify what these comments refer to?  Are they tests which you hoped would work but don't?

Reply:
Writing `attribute [-norm_num] ...` a second time should give the error that the identifier does not have the `norm_num` attribute, as it has already been erased. This comment tries to say that if you uncomment those lines of code, it should (and does) give that error—I'll clarify it so that it actually says that :) 

Or maybe there's a way to use `successIfFail` here?

Reply:
Clarified! I couldn't find a way to make `successIfFail` work, though.